R/update-news.R

```diff
@@ -354,6 +355,35 @@ default_type <- function() {
   "Uncategorized"
 }
 
+capitalize_description <- function(df) {
+  df$description
+
+  # leave code alone
+  if (substr(df$description, 1, 1) == "`") {
+    return(df)
+  }
+
+  # leave package name alone
+  pkg_name <- desc::desc_get("Package")
+  start_with_pkg <- (grepl(sprintf("^%s ", pkg_name), df$description) ||
+    grepl(sprintf("^%s's", pkg_name), df$description))
```

Comment:
`"^%s(?: |'s)"` ? Extract regex to separate variable? And perhaps combine with regex above?

Reply:
In the end I realized the "code" regex was useless as `toupper()` will not touch those anyway.